--- origsrc/openal-soft-1.13/CMakeLists.txt	2011-02-15 14:05:40.000000000 -0600
+++ src/openal-soft-1.13/CMakeLists.txt	2011-11-27 00:49:39.269851400 -0600
@@ -397,7 +397,7 @@ IF(DSOUND)
     ENDIF()
 ENDIF()
 
-IF(HAVE_WINDOWS_H)
+IF(HAVE_LIBWINMM)
     IF(WINMM)
         CHECK_INCLUDE_FILES("windows.h;mmsystem.h" HAVE_MMSYSTEM_H -D_WIN32_WINNT=0x0500)
         IF(HAVE_MMSYSTEM_H AND HAVE_LIBWINMM)
--- origsrc/openal-soft-1.13/Alc/portaudio.c	2010-12-04 19:36:20.000000000 -0600
+++ src/openal-soft-1.13/Alc/portaudio.c	2011-08-08 11:57:55.480527200 -0500
@@ -72,6 +72,8 @@ void *pa_load(void)
     const char *str;
 #if defined(__APPLE__) && defined(__MACH__)
 # define PALIB "libportaudio.2.dylib"
+#elif defined(__CYGWIN__)
+# define PALIB "cygportaudio-2.dll"
 #else
 # define PALIB "libportaudio.so.2"
 #endif
--- origsrc/openal-soft-1.13/Alc/pulseaudio.c	2011-02-08 01:01:39.000000000 -0600
+++ src/openal-soft-1.13/Alc/pulseaudio.c	2011-11-26 23:53:24.199808400 -0600
@@ -183,6 +183,8 @@ void *pulse_load(void) //{{{
         const char *err;
 #if defined(__APPLE__) && defined(__MACH__)
         pa_handle = dlopen("libpulse.0.dylib", RTLD_NOW);
+#elif defined(__CYGWIN__)
+        pa_handle = dlopen("cygpulse-0.dll", RTLD_NOW);
 #else
         pa_handle = dlopen("libpulse.so.0", RTLD_NOW);
 #endif
