--- origsrc/openal-soft-1.13/CMakeLists.txt	2011-02-15 14:05:40.000000000 -0600
+++ src/openal-soft-1.13/CMakeLists.txt	2011-11-27 00:49:39.269851400 -0600
@@ -397,7 +397,7 @@ IF(DSOUND)
     ENDIF()
 ENDIF()
 
-IF(HAVE_WINDOWS_H)
+IF(HAVE_LIBWINMM)
     IF(WINMM)
         CHECK_INCLUDE_FILES("windows.h;mmsystem.h" HAVE_MMSYSTEM_H -D_WIN32_WINNT=0x0500)
         IF(HAVE_MMSYSTEM_H AND HAVE_LIBWINMM)
--- origsrc/openal-soft-1.15.1/Alc/backends/portaudio.c	2012-12-11 14:35:55.000000000 -0600
+++ src/openal-soft-1.15.1/Alc/backends/portaudio.c	2013-04-10 01:15:40.215727100 -0500
@@ -67,6 +67,8 @@ static ALCboolean pa_load(void)
     {
 #ifdef _WIN32
 # define PALIB "portaudio.dll"
+#elif defined(__CYGWIN__)
+# define PALIB "cygportaudio-2.dll"
 #elif defined(__APPLE__) && defined(__MACH__)
 # define PALIB "libportaudio.2.dylib"
 #elif defined(__OpenBSD__)
--- origsrc/openal-soft-1.15.1/Alc/backends/pulseaudio.c	2012-12-11 14:35:55.000000000 -0600
+++ src/openal-soft-1.15.1/Alc/backends/pulseaudio.c	2013-04-10 01:16:02.328991900 -0500
@@ -234,6 +234,8 @@ static ALCboolean pulse_load(void)
     {
 #ifdef _WIN32
 #define PALIB "libpulse-0.dll"
+#elif defined(__CYGWIN__)
+#define PALIB "cygpulse-0.dll"
 #elif defined(__APPLE__) && defined(__MACH__)
 #define PALIB "libpulse.0.dylib"
 #else
