--- origsrc/openal-soft-1.10.622/CMakeLists.txt	2009-11-08 22:17:12.000000000 -0600
+++ src/openal-soft-1.10.622/CMakeLists.txt	2009-12-11 04:34:54.025041200 -0600
@@ -119,7 +119,7 @@ ELSE()
         FORCE)
 
     # Set visibility options if available
-    IF(NOT WIN32)
+    IF(NOT WIN32 AND NOT CYGWIN)
         CHECK_C_SOURCE_COMPILES("int foo() __attribute__((destructor));
                                  int main() {return 0;}" HAVE_GCC_DESTRUCTOR)
 
@@ -212,8 +212,10 @@ IF(DLOPEN)
     ENDIF()
 ENDIF()
 
+if(WIN32)
 # Check if we have Windows headers
 CHECK_INCLUDE_FILE(windows.h HAVE_WINDOWS_H -D_WIN32_WINNT=0x0500)
+ENDIF()
 IF(NOT HAVE_WINDOWS_H)
     CHECK_FUNCTION_EXISTS(gettimeofday HAVE_GETTIMEOFDAY)
     IF(NOT HAVE_GETTIMEOFDAY)
