--- openal-0.0.8-orig/configure.ac	2006-02-11 03:36:55.000000000 -0600
+++ openal-0.0.8/configure.ac	2006-05-12 18:01:13.426375000 -0500
@@ -648,7 +648,7 @@
   openal_backend_bsd_support=disabled
 else
   case "$target" in
-    *bsd*) openal_backend_bsd_support=static ;;
+    *bsd*|*cygwin*) openal_backend_bsd_support=static ;;
     *) openal_backend_bsd_support=none
        if test "x$enable_bsd" = xyes; then
          AC_MSG_WARN([--enable-bsd ignored because this is no BSD build.])
--- openal-0.0.8-orig/src/backends/alc_backend_bsd.c	2006-01-05 09:11:20.000000000 -0600
+++ openal-0.0.8/src/backends/alc_backend_bsd.c	2006-05-12 18:27:46.379500000 -0500
@@ -9,7 +9,7 @@
  */
 #include "al_siteconfig.h"
 
-#include <AL/al.h>
+#include <AL/alext.h>
 #include <assert.h>
 #include <fcntl.h>
 #include <sys/soundcard.h>
